docs/references/system_requirements/global.md

```diff
@@ -11,7 +11,6 @@ For the Epinio server, and related deployments we recommend to consider the foll
 - Supported operating systems
   - Linux
     - K3s, K3d
```

Comment:
```suggestion
    - K3s, K3d, RKE2
```

Reply:
Hm. Maybe also Rancher Desktop, as is done in the Windows item just below.
RKE2 is full-blown server ...
That said I will make the names all links to the respective installation pages.

Reply:
Done with commit 75499a4b047ab70364b0ab5a0896642b4ea9c59b

Reply:
Better, thanks